examples/with-docker/package.json

```diff
@@ -2,8 +2,7 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
-    "start": "next start -p ${PORT:=3000}"
```

Comment:
Removed unused script for clarity - the Dockerfile contains the entrypoint in this example